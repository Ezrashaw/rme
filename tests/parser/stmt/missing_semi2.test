let x = 1 + 1
let y = 2 + 2;