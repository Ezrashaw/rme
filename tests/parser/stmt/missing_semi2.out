[1;3;31merror[0m: expected `;` but found `let`
[38;5;244;1m  |[0m 
[38;5;244;1m2 |[0m let y = 2 + 2;
[38;5;244;1m  |[0m [1;96m^^^[0m
